scripts/components/OwnerRepairRequestStatus.jsx

```diff
@@ -0,0 +1,121 @@
+var React = require('react');
+var moment = require('moment');
+var Api = require('../utils/Api.js');
+var MuiContextified = require('./MuiContextified.jsx');
+var mui = require('material-ui');
+var SelectField = mui.SelectField;
+var RaisedButton = mui.RaisedButton;
+var Dialog = mui.Dialog;
+
+var OwnerRepairRequestStatus = React.createClass({
+  getInitialState() {
+    return {
+      repairStatus: '',
+    }
+  },
+  
+  propTypes: {
+    statusChanged: React.PropTypes.func,
+    propertyId: React.PropTypes.number,
+    requestId: React.PropTypes.number,
+  },
+
+  onButtonClick() {
+    this.refs.dialog.show();
+  },
+
+  // Update select menu on change.
+  onSelectChange(event, selectedUserTypeIndex) {
+    var repairStatus = status[selectedUserTypeIndex].name;
+    this.setState({ 
+      repairStatus: repairStatus,
+    });
+  },
+
+  // Handle the form submission event when the user updates repair request.
+  onSubmit(event) {
+
+    // API call to update repair request status
+    var propertyId = this.props.propertyId;
+    var requestId = this.props.requestId;
+    Api.updateRepairRequest({
+      data: {
+        repairStatus: this.state.repairStatus,
+        requestId: this.props.requestId,
+        propertyId: this.props.propertyId,
+      },
+      callback: (err, response) => {
+        if (err) {
+          return;
+        }
+        // Clear 
+        this.setState({
+          repairStatus: '',
+        });
+        this.props.statusChanged();
+        this.refs.dialog.dismiss();
+      }
+    });
+  },
+
+  render() {
+    var { repairStatus } = this.state;
+    var errorMessage;
+    var standardActions = [
+      { text: 'Cancel' },
+      { text: 'Change Status', onTouchTap: this.onSubmit, ref: 'submit' }
+    ];
+    return (
+      <div style={style.formContainer}>
+        <RaisedButton label="Change Status"
+                      primary={true}
+                      onClick={this.onButtonClick} />
+        <Dialog
+          title="Change Status"
+          actions={standardActions}
+          actionFocus="submit"
+          modal={this.state.modal}
+          ref="dialog">
+          <div style={style.form}>
+            <div style={style.error}> { errorMessage } </div>
```

Comment:
style.error is not defined on this page. Rather than adding this style here, instead use the <ErrorMessage> component I made.